The kernel can be configured to be compiled for multiple devices,
this patch does that configuration.
---

diff --git a/arch/arm64/boot/dts/Makefile b/arch/arm64/boot/dts/Makefile
index 96e7219d8..92a07a684 100644
--- a/arch/arm64/boot/dts/Makefile
+++ b/arch/arm64/boot/dts/Makefile
@@ -5,6 +5,12 @@ dtb-$(CONFIG_SOC_EXYNOS8890) += exynos8890-espresso8890.dtb exynos8890-espresso8
 dtb-$(CONFIG_SOC_EXYNOS8890) += exynos8890-evt1-smdk8890.dtb
 dtb-$(CONFIG_SOC_EXYNOS7870) += exynos7870-smdk7870.dtb exynos7870-universal7870.dtb exynos7870-universal7870_rev01.dtb
 
+dtb-$(CONFIG_SOC_EXYNOS7870) += exynos7870-j7xelte_eur_open_00.dtb \
+exynos7870-j7xelte_eur_open_01.dtb \
+exynos7870-j7xelte_eur_open_02.dtb \
+exynos7870-j7xelte_eur_open_03.dtb \
+exynos7870-j7xelte_eur_open_04.dtb
+
 targets += dtbs
 
 DTB_NAMES := $(subst $\",,$(CONFIG_BUILD_ARM64_APPENDED_DTB_IMAGE_NAMES))
